### PR TITLE
More UWLogin specific instructions for parameter files

### PR DESCRIPTION
**Pull request by truggles**

### CODE REVIEW

File: NtupleTools/test/README.rst [hunk 1]
```diff
@@ -42,8 +42,12 @@ zero or one) are::
     paramFile=''            - analysis-specific parameter file defining cuts and extra 
                               ntuple variables. Note that when submitting to CONDOR from
                               UWLogin, the path should be specified starting with
-                              'CMSSW_7_X_Y_pZ/src/...'. If the file cannot be found, 
-                              a default is used and the job will not crash.
+                              'CMSSW_7_X_Y_pZ/src/...'. If you have renamed your CMSSW_BASE, 
+                              you will need to find the default name in 'CMSSW_X_X_X formate. 
+                              If the file cannot be found, a default is used and the job will not crash.
+                              Additionally, on UWLogin, you will need to include 
+                              "--extra-usercode-files 'src/FinalStateAnalysis/NtupleTools/python/parameters'"
```
Comment (dntaylor) [inline NtupleTools/test/README.rst]: has this been added to submit jobs?

**Comment by nwoods** [comment]
We could make it a permanent feature if you want, @dntaylor.

**Comment by dntaylor** [comment]
If it is necessary to run on uwlogin, it should be included in submit_job.py. Add an `if host is uwlogin.cern.ch` test after this and add in the `--extra-usercode-files` line. https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/Utilities/scripts/submit_job.py#L103

**Comment by truggles** [comment]
I'll add it soon and repull

**Pull request merged**